src/main/java/DetermineCourseCompletion.java

```diff
@@ -43,4 +44,24 @@ public static LocalDateTime getCourseCompletionDate(LocalDateTime startDate, int
         }
         return endDate;
     }
+
+    public static LocalDateTime determineEndDateSkippingWeekends(LocalDateTime startDate, int coursesDuration) {
```

Comment:
could you implement this logic inside the `getCourseCompletionDate` method? 
thus, we ensure existing tests are passing for `getCourseCompletionDate`